<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1720</width>
                <height>1100</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>1720</width>
                <height>1100</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralWidget">
            <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayout" stretch="280,720">
                        <item>
                            <layout class="QFormLayout" name="formLayout">
                                <item row="4" column="0">
                                    <widget class="QLabel" name="airlineLabel">
                                        <property name="minimumSize">
                                            <size>
                                                <width>30</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>Airline:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="2" column="0">
                                    <widget class="QLabel" name="fromLabel">
                                        <property name="minimumSize">
                                            <size>
                                                <width>30</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>From:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="3" column="0">
                                    <widget class="QLabel" name="toLabel">
                                        <property name="minimumSize">
                                            <size>
                                                <width>30</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>To:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="5" column="1">
                                    <widget class="QPushButton" name="pushButton">
                                        <property name="text">
                                            <string>Search</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="4" column="1">
                                    <widget class="QLineEdit" name="AirlineSearch">
                                        <property name="text">
                                            <string/>
                                        </property>
                                    </widget>
                                </item>
                                <item row="3" column="1">
                                    <widget class="QLineEdit" name="ToSearch">
                                        <property name="text">
                                            <string/>
                                        </property>
                                    </widget>
                                </item>
                                <item row="2" column="1">
                                    <widget class="QLineEdit" name="FromSearch">
                                        <property name="text">
                                            <string/>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
                                <item>
                                    <widget class="DrawableMapWidget" name="map" native="true">
                                        <property name="minimumSize">
                                            <size>
                                                <width>1440</width>
                                                <height>720</height>
                                            </size>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QListWidget" name="flighttable">
                                        <property name="minimumSize">
                                            <size>
                                                <width>720</width>
                                                <height>360</height>
                                            </size>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1720</width>
                    <height>22</height>
                </rect>
            </property>
        </widget>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <customwidgets>
        <customwidget>
            <class>DrawableMapWidget</class>
            <extends>QWidget</extends>
            <header>drawablemapwidget.h</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
